apps/global_preferences.h

```diff
@@ -36,6 +36,8 @@ class GlobalPreferences {
   void setAutocomplete(bool autocomple) { m_autoComplete = autocomple; }
   bool syntaxhighlighting() const { return m_syntaxhighlighting; }
   void setSyntaxhighlighting(bool syntaxhighlight) { m_syntaxhighlighting = syntaxhighlight; }
+  bool clearshift() const { return  m_clearkshift; }
+  void setclearshift(bool clearalphashift) {  m_clearkshift = clearalphashift; }
```

Comment:
Can you explain what the `k` in `m_clearkshift` means? Also, avoid doubles spaces

```suggestion
  bool clearshift() const { return m_clearkshift; }
  void setclearshift(bool clearalphashift) { m_clearkshift = clearalphashift; }
```